### DISCUSSION TITLE
How to auto update plugins ?

### DISCUSSION THREAD

**Discussion by OriLiMu** [LazyVim/LazyVim#1911] [Q&A] [answered]
There will be a message box to warn you to update plugin if available. How to make it auto update? 
I write a crontab script to make it. How to auto update just in nvim

**Comment by dpetka2001** [comment, marked as answer]
See this [issue](https://github.com/folke/lazy.nvim/issues/702) in `lazy.nvim` repo.

**Reply by andresatierf** [reply]
I added the following to my `autocmds.lua` but nothing happens. I does not seem to trigger. If I change the event to something else, like `BufEnter` the callback is run.

```lua
-- `autocmd` is `vim.api.nvim_create_autocmd` and `augroup` is the same as in the link you provided
autocmd("VimEnter", {
  group = augroup("autoupdate"),
  callback = function()
    require("lazy").update({ show = false })
  end,
})
```


Do you have any insights?

**Reply by RoccoRakete** [reply]
It needs to go into `options.lua`.

See https://github.com/folke/lazy.nvim/issues/702#issuecomment-2308894450

**Reply by DanOpcode** [reply]
For me I think it works by updating `.config/nvim/init.lua` with:

```lua
require("config.lazy")
vim.api.nvim_create_autocmd("VimEnter",{callback=function()require"lazy".update({show = false})end})
```
